libs/auth-api-lib/src/lib/entities/models/client.model.ts

```diff
@@ -37,6 +37,24 @@ export class Client extends Model<Client> {
   })
   clientId: string
 
+  @Column({
+    type: DataType.STRING,
+    allowNull: true,
+  })
+  @ApiProperty({
+    example: '1234567890',
+  })
+  nationalId: string
```

Comment:
Suggestion: maybe we can restrict null here, even though the db allows nulls for now?